sdk/resourcemanager/Azure.ResourceManager.Core/src/Generated/SubscriptionOperations.cs

```diff
@@ -97,6 +97,15 @@ public virtual LocationContainer GetLocations()
             return new LocationContainer(this);
         }
 
+        /// <summary>
+        /// Gets the tags container under this subscription.
+        /// </summary>
+        /// <returns> The tags container. </returns>
+        public virtual TagsContainer GetTags()
+        {
+            return new TagsContainer(new ClientContext(ClientOptions, Credential, BaseUri, Pipeline), Id.SubscriptionId);
```

Comment:
this would typically take in Id itself, then you don't need to reconstruct it later in the container ctor